crates/kubelet/Cargo.toml

```diff
@@ -26,3 +26,5 @@ tokio  = { version = "0.2.13" }
 kube = { version = "0.28.0"  }
 k8s-openapi = { version = "0.7.1", default-features = false, features = ["v1_17"] }
 chrono = { version = "0.4", features = ["serde"] }
+clap = { git = "https://github.com/clap-rs/clap", features = ["wrap_help"] }
```

Comment:
Will users universally configure from the command line? If not, we should think about putting this behind a feature flag.

Reply:
I like this idea. Let me figure out how to do it, because I haven't before